ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -1577,6 +1585,14 @@ IE_GPU.onnx_model_gather_elements_int32_axis_0
 IE_GPU.onnx_model_gather_elements_int8_axis_1
 IE_GPU.onnx_model_gather_elements_float_3D_axis_2
 
+# Roll is not implemented yet for CPU, GPU 
```

Comment:
Please do not duplicate test disabling for CPU and GPU if you want to disable on both CPU and GPU just write the name of the test, e.g. roll_2d_input